src/loki_alert_rules/error_level_logs_found.rule

```diff
@@ -0,0 +1,7 @@
+alert: ErrorLevelLogsFound
+expr: rate({%%juju_topology%%} |= "level=error"[1m]) > 0
+for: 45s
```

Comment:
45 seconds seems like quite a short window especially that we look at logs over the last minute. How about a longer time period, like 10m like in the example here? https://grafana.com/docs/loki/latest/alert/#example